Backend/Remora.Discord.API/Json/Converters/Internal/DiscordPermissionSetConverter.cs

```diff
@@ -45,24 +45,24 @@ JsonSerializerOptions options
         switch (reader.TokenType)
         {
             case JsonTokenType.String:
-            {
```

Comment:
Case braces shall be included and shall not be indented beyond the level of the case itself; revert.